tests/conftest.py

```diff
@@ -72,26 +72,37 @@ def polarion_testsuite_properties(record_testsuite_property):
         )
 
 
-def cluster_teardown(log_level="DEBUG"):
+def cluster_teardown(deployer, log_level="DEBUG"):
```

Comment:
I think this is not need to be in the function and can be directly called from deployer like deployer.destroy_cluster()

Reply:
Done